### PR TITLE
EOS-23137: Setup passwordless-ssh for nodeadmin user

### PR DESCRIPTION
**Pull request by SomwanshiAnjali**
Signed-off-by: Anjali Somwanshi <anjali.somwanshi@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: api/python/provisioner/inputs.py [hunk 1]
```diff
@@ -436,9 +436,9 @@ class PillarInputBase(PillarItemsAPI):
         metadata={
             METADATA_ARGPARSER: {
                 'help': 'pillar value',
-                'type': functools.partial(
-                    AttrParserArgs.value_from_str, v_type='json'
-                )
+                #'type': functools.partial(
```
Comment (nitin-seagate) [inline api/python/provisioner/inputs.py:439]: Why do we delete this?

File: lr-cli/cortx_setup/commands/cluster/create.py [hunk 2]
```diff
@@ -256,67 +258,87 @@ def run(self, **kwargs):
...
+            )
+
+            machine_id = self.provisioner.grains_get("machine_id")[node_id]["machine_id"]
+            enclosure_id = self.provisioner.grains_get("enclosure_id")[node_id]["enclosure_id"]
             if enclosure_id:
                 if not machine_id in enclosure_id:   # check if the system is VM or HW
                     self.logger.debug(f"Setting time on enclosure with server & timezone")
-                    NodePrepareTime().set_enclosure_time()
-            StatesApplier.apply( ['components.system.config.sync_salt'] ,targets=ALL_MINIONS)
+                    #NodePrepareTime().set_enclosure_time()
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/cluster/create.py:313]: Remove commented code...

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
Where do we set permissions for ConfStore JSON file?

File: api/python/provisioner/inputs.py (hunk 1 above)
Comment (SomwanshiAnjali) [reply api/python/provisioner/inputs.py:439]: This code is failing and we don't need this for recent changes.

File: lr-cli/cortx_setup/commands/cluster/create.py (hunk 2 above)
Comment (SomwanshiAnjali) [reply lr-cli/cortx_setup/commands/cluster/create.py]: Addressed.

File: api/python/provisioner/inputs.py (hunk 1 above)
Comment (nitin-seagate) [reply api/python/provisioner/inputs.py:439]: If its not required delete this block (do not comment)

**Review by nitin-seagate — APPROVED** [review]

File: api/python/provisioner/inputs.py (hunk 1 above)
Comment (SomwanshiAnjali) [reply api/python/provisioner/inputs.py]: Addressed

File: node_cli/schema/cluster.json [hunk 3]
```diff
@@ -108,6 +108,11 @@
             "type": "str",
             "help": "Management vip"
         },
+        {
```
Comment (nitin-seagate) [inline node_cli/schema/cluster.json:111]: Please accept password as prompt input instead of plaintext password?

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — COMMENTED** [review]
LGTM

**Review by nitin-seagate — APPROVED** [review]

**Pull request merged**